src/13_file_io.py

```diff
@@ -9,10 +9,30 @@
 # Print all the contents of the file, then close the file
```

Comment:
The code works fine, good job with that. However, I would suggest creating a function for it instead of writing separate lines of code.